docs/products/postgresql/howto/migrate-db-to-aiven-via-console.rst

```diff
@@ -0,0 +1,308 @@
+
+Prerequisites
+-------------
+
+* Logical replication is enabled on the source database.
+* You have superuser permissions on the source database.
+* Source database's hostname or IP address are :doc:`accessible from the public Internet </docs/platform/howto/public-access-in-vpc>`.
+* You have the source database's credentials and reference data
+  
+  * Public hostname or connection string, or IP address used to connect to the database
+  * Port used to connect to the database
+  * Username (for a user with superuser permissions)
+  * Password
+
+* Firewalls protecting the source database and the target databases are open to allow the traffic and connection between the databases (update or disable the firewalls temporarily if needed).
+
+Pre-configure the source
+------------------------
+
+* Allow remote connections.
+
+  Check that your database allows all remote connections, use ``psql`` to run the following query:
+
+  .. code-block:: bash
+
+     SHOW listen_addresses;
+
+  If enabled, you can expect the following output (with ``listen_addresses`` set to ``*``):
+
+  .. code-block:: bash
+
+      listen_addresses
+      -----------
+      *
+      (1 row)
+
+  If the command line returns something different, enable remote connections for your database with the following query:
+
+  .. code-block:: bash
+
+     ALTER SYSTEM SET listen_addresses = '*';
+
+* Change your IPv4 local connection to ``0.0.0.0/0`` to allow all incoming IP addresses.
+
+  Find the ``pg_hba.conf`` configuration file using the following query:
+
+  .. code-block:: bash
+
+     SHOW hba_file;
+
+  Open ``pg_hba.conf`` in a text editor of your choice, for example, Visual Studio Code:
+
+  .. code-block:: bash
+
+     code pg_hba.conf
+
+  Under ``IPv4 local connections``, find and replace the IP address with ``0.0.0.0/0``.
+
+  .. code-block:: bash
+
+     # TYPE  DATABASE        USER            ADDRESS                 METHOD
+     
+     # IPv4 local connections:
+     host    all             all             0.0.0.0/0               md5
+     # IPv6 local connections:
+     host    all             all             ::/0                    md5 
+
+  .. seealso::
+   
+     For more details on the configuration file's syntax, see `The pg_hba.conf File <https://www.postgresql.org/docs/14/auth-pg-hba-conf.html>`_.
+
+* Enable logical replication.
+
+  For cloud-hosted databases, the logical replication is usually enabled by default, while databases hosted on-premises can have the logical replication not enabled.
+
+  Check that the logical replication is enabled, use ``psql`` to run the following query:
+
+  .. code-block:: bash
+
+     SHOW wal_level;
+
+  .. code-block:: bash
+     :caption: Expected output if enabled
+
+     wal_level
+     -----------
+     logical
+     (1 row)
+
+  If the command prompt returns something different, enable the logical replication in your database by setting ``wal_level`` to ``logical``:
```

Comment:
I think we require _at least_ `logical` (I remember reading that somewhere in the code) so it may be worth trying that out or double checking that with the documentation.

Reply:
There's nothing more after `logical` [https://www.postgresql.org/docs/current/runtime-config-wal.html](https://www.postgresql.org/docs/current/runtime-config-wal.html)
It's good for me!